### PR TITLE
Add CRNA for development

### PR DESCRIPTION
**Pull request by satya164**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -87,7 +87,7 @@ project.ext.vectoricons = [
 ]
 
 apply from: "../../node_modules/react-native/react.gradle"
-apply from: "../../node_modules/react-native-vector-icons/fonts.gradle"
+apply from: "./fonts.gradle"
```
Comment (satya164) [inline android/app/build.gradle]: The exponent fork for vector icons doesn't include this file

File: src/components/Home.js [hunk 2]
```diff
@@ -8,7 +10,34 @@ import routeMapper from './Navigation/routeMapper';
...
+  };
+
+  componentWillMount() {
+    this._bootstrap();
+  }
+
+  _bootstrap = async () => {
+    await Font.loadAsync({
+      /* $FlowFixMe */
+      Montserrat: require('../../assets/fonts/Montserrat.otf'),
```
Comment (satya164) [inline src/components/Home.js]: This gives error when run without exponent

File: exp.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "iconUrl": "https://s3.amazonaws.com/exp-us-standard/icons/pocket-gear.png",
+    "hideExponentText": false
+  },
+  "androidStatusBarColor": "#000000",
+  "packagerOpts": {
+    "assetExts": [
+      "ttf", "otf"
+    ]
+  }
+}
```
Comment (satya164) [inline exp.json]: Figure out a way to,

1. Integrate Fabric for Android and iOS (without having to commit the build secret)
2. Specify separate icons for iOS and Android apps locally
3. Set the accent color to black for Android
4. Set statusbar content color to dark on Android

**Comment by satya164** [comment]
cc @brentvatne

File: exp.json (hunk 3 above)
Comment (brentvatne) [inline exp.json]: more context for other Exponent people who look at this

2. We need to have an iconUrl for iOS and for Android
3. We support primary color but accent color (default spinner color etc) is useful
4. I think this is a fairly new API, @satya164 could you try to get something upstream to React Native for this? As for Exponent, we should deprecate our existing android status bar config from exp.json

File: android/app/build.gradle (hunk 1 above)
Comment (brentvatne) [inline android/app/build.gradle]: what is this file for?

File: exp.json (hunk 3 above)
Comment (jesseruder) [inline exp.json]: I've added tickets for these in our internal Trello. Thanks!

Comment (satya164) [inline exp.json]: @jesseruder Thank you

**Comment by brentvatne** [comment]
sick bro

**Pull request merged**